app/views/taxons/show_a.html.erb

```diff
@@ -38,25 +38,25 @@
   <nav role="navigation" class="taxon-page__grid">
     <div class="full-page-width-wrapper">
       <%= render "govuk_publishing_components/components/heading", {
-        text: t('taxons.sub_topics'),
-        heading_level: 2,
-        font_size: 19,
-        margin_bottom: 2
+          text: t('taxons.sub_topics'),
+          heading_level: 2,
+          font_size: 19,
+          margin_bottom: 2
       } %>
       <ol class="taxon-page__grid-wrapper">
         <% presented_taxon.child_taxons.each_with_index do |child_taxon, index| %>
           <li class="taxon-page__grid-item">
             <h3 class="taxon-page__grid-heading">
               <%= link_to(
-                child_taxon.title,
-                child_taxon.base_path,
-                data: presented_taxon.options_for_child_taxon(index: index)
-              )%>
+                      child_taxon.title,
```

Comment:
Indenting

Reply:
Indenting comments will also probably apply to `show_c` as it's been copied from `show_a`

Reply:
👍